README.md

```diff
@@ -73,16 +70,16 @@ You can check the status of the nodes using the EC2 Dashboard or by running the
 
 The `muchos setup` command will set up your cluster and start Hadoop, Zookeeper, & Accumulo.  It
 will download release tarballs of Fluo, Accumulo, Hadoop, etc. The versions of these tarballs are
-specified in `muchos.props` and can be changed if desired.
+specified in [muchos.props] and can be changed if desired.
 
 Optionally, Muchos can setup the cluster using an Accumulo or Fluo tarball that is placed in the
 `conf/upload` directory of Muchos. This option is only necessary if you want to use an unreleased
 version of Fluo or Accumulo. Before running the `muchos setup` command, you should confirm that the
-version and SHA-256 hash of your tarball matches what is set in `conf/muchos.props`. Run the command
+version and SHA-256 hash of your tarball matches what is set in [muchos.props]. Run the command
 `shasum -a 256 /path/to/tarball` on your tarball to determine its hash.
 
 The `muchos setup` command will install and start Accumulo, Hadoop, and Zookeeper.  The optional 
-services below will only be set up if configured in the [nodes] section of `muchos.props`:
+services below will only be set up if configured in the [nodes] section of [muchos.props]:
```

Comment:
I don't see a corresponding link for `[nodes]`

Reply:
It's a section in muchs.props.  I made the text `[nodes]` to be more explicit